scala2.13/udf-compiler/pom.xml

```diff
@@ -130,9 +130,6 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <skipTests>true</skipTests>
-                </configuration>
```

Comment:
Scala 2.13 pom was edited directly rather than generated, causing all the 2.13 builds to fail.

I'm not sure why this change was made, since the comment just above it seems to still apply:
```
            <!-- disable surefire as we are using scalatest only -->
```

Reply:
Updated here: https://github.com/NVIDIA/spark-rapids/pull/11553/commits/d26258b2299100e2ed78157ac420b4f84ae213ed